src/admin-toolkit-ui/index.tsx

```diff
@@ -311,16 +191,10 @@ export async function initializeAdminData(
   console.log('initializeAdminData')
   if (!adminDataInitialized) {
     console.log('initializeAdminData - not initialized')
-    const { VideoControlState, TextAnnouncements } = getComponents(engine)
-
-    // Initialize scene data
-    await Promise.all([initSceneDeployment(), initSceneOwners()])
-
+    const { TextAnnouncements, VideoControlState } = getComponents(engine)
```

Comment:
the component `VideoControlState` was created to simulate the NetworkEntity for Video Players.

as the VideoPlayer now is supported by the `NetworkEntity`, I think the VideoControlState is no longer required

Reply:
yessssssss, but i'm using it now for the livekit stream key and to sync that key through the network